src/ast/parse.rs

```diff
@@ -370,21 +370,33 @@ fn command(ctx: &Context) -> Res<Command> {
                 map(option(text(":unextractable")), |x, _| x.is_some()),
                 map(option(sequence(text(":on_merge"), list(action))), snd),
                 map(option(sequence(text(":merge"), expr)), snd),
```

Comment:
This doesn't seem right- shouldn't `:merge` be required for functions?
Otherwise, you can use a constructor (or panic for primitives).
Perhaps this should be a follow-up PR.

Reply:
This is not enforced right now. Functions without a merge function will throw an error when set with different values.

Reply:
Ah okay- so if we don't default to `union` even for eq-able types? That was the old behavior that was confusing